index.html

```diff
@@ -3177,6 +3177,273 @@ <h3>Definition of Roles</h3>
+        <div class="role">
+          <rdef>doc-pagenum</rdef>
+          <div class="role-description">
+            <p>A visible indicator of the page number.</p>
+            <p>The doc-pagenum role MUST be used on visible page numbers present in page headers and page footers. See <rref>doc-pageheader</rref> or <rref>doc-pagefooter</rref> for examples.</p>
+            <p>The current page number SHOULD also be provided as a name on page break elements. See the examples provided in <rref>doc-pagebreak</rref> for specifics.</p>
+          </div>
+          <table class="role-features">
+            <caption>Characteristics:</caption>
+            <thead>
+              <tr>
+                <th scope="col">Characteristic</th>
+                <th scope="col">Value</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <th class="role-abstract-head" scope="row">Is Abstract:</th>
+                <td class="role-abstract"> </td>
+              </tr>
+              <tr>
+                <th class="role-parent-head" scope="row">Superclass Role:</th>
+                <td class="role-parent"><rref>section</rref></td>
+              </tr>
+              <tr>
+                <th class="role-children-head" scope="row">Subclass Roles:</th>
+                <td class="role-children"> </td>
+              </tr>
+              <tr>
+                <th class="role-base-head" scope="row">Base Concept:</th>
+                <td class="role-base"> </td>
+              </tr>
+              <tr>
+                <th class="role-related-head" scope="row">Related Concepts:</th>
+                <td class="role-related">EPUB <a
+                    href="https://idpf.github.io/epub-vocabs/structure/#chapter">chapter</a>
+                  [[EPUB-SSV]]</td>
+              </tr>
+              <tr>
+                <th class="role-scope-head" scope="row">Required Context Role:</th>
+                <td class="role-scope"><rref>doc-pagefooter</rref> or <rref>doc-pageheader</rref></td>
+              </tr>
+              <tr>
+                <th class="role-mustcontain-head" scope="row">Required Owned Elements:</th>
+                <td class="role-mustcontain"></td>
+              </tr>
+              <tr>
+                <th class="role-required-properties-head">Required States and Properties:</th>
+                <td class="role-required-properties"> </td>
+              </tr>
+              <tr>
+                <th class="role-properties-head" scope="row">Supported States and Properties:</th>
+                <td class="role-properties"> </td>
+              </tr>
+              <tr>
+                <th class="role-inherited-head" scope="row">Inherited States and Properties:</th>
+                <td class="role-inherited"> </td>
+              </tr>
+              <tr>
+                <th class="role-namefrom-head" scope="row">Name From:</th>
+                <td class="role-namefrom">author</td>
```

Comment:
No, and in fact, most of these DPUB roles allow name from author, and I'm not sure which have been really thought about.

Reply:
I think that name from "prohibited" was added to ARIA after DPUB-ARIA was created. There wasn't a concept of not allowing a name.

I've added "prohibited" for doc-pagenum.

Reply:
Another option is "prohibited", which is new in ARIA 1.2, so that's probably why it wasn't part of the design of these roles.
Examples of roles where naming is prohibited: https://w3c.github.io/aria/#namefromprohibited

Reply:
I think we should prohibit naming of all 3 new roles. And in fact, I think we should open up an issue to add prohibited to other roles where it makes sense too.

Reply:
Opened https://github.com/w3c/dpub-aria/issues/30

Reply:
👍 marking this conversation resolved.